### ISSUE TITLE
Rancher 2.6.8 UI throwing constant "Websocket Disconnected" errors

### ISSUE THREAD

**Issue by gaktive** [rancher/dashboard#6785]
Internal reference: SURE-5185

Issue description:

After upgrading to Rancher 2.6.8, a user started seeing these errors continuously in the UI:

```
Websocket Disconnected 
The connection to wss://SERVER.DOMAIN.TLD/v3/subscribe closed unexpectedly 
Tue, Aug 30 2022 10:14:49 am. 
Retrying...
```

Some of their RKE downstream clusters did not upgrade. ~~There was a script that another environment had used for similar issue on AKS. This was used here with a slight change to make it work for RKE. After stopping Rancher, running the script, the clusters were upgraded and healthy.~~ (Irrelevant section struck out upon follow-up.)

But the UI continued to show that websocket error continuously every few seconds. Looks like this is because of: https://github.com/rancher/dashboard/issues/5093#issuecomment-1188467756

There doesn't seem to be any other issues with the Rancher UI and all the downstream clusters are healthy. We can't tell where the errors are coming from... Rancher? Downstream cluster agents? The web browser? The customer's networking (i.e. Load Balancer)?

Business impact:
A constant stream of errors in the UI is a real hindrance for all users and the user would like a way to silence/disable these growls since these errors and find out why it's happening.

Troubleshooting steps:
We tried refreshing the browser, logging out and logging back in, using a local user instead of LDAP, and using a direct connection without VPN. All of these still caused the errors to pop up continuously in the UI. 

Actual behavior:
Constant stream of websocket errors in the UI.

Expected behavior:
No websocket errors in the UI.

**Comment by gaktive** [comment]
At the very least, we need a way to disable alerts to allow users a chance to diagnose what's going on since we're impeding troubleshooting. As for what's causing the issue that's happening in the backend, some log analysis may be needed.

**Comment by cbron** [comment]
Websocket errors are very common, and can occur for many reasons like network instability. I would reconsider this being on by default. We can have wider discussion about it.

**Comment by gaktive** [comment]
Possible connection to https://github.com/rancher/dashboard/issues/5997

Jira showing a lot more reports of this behaviour.

**Comment by jbilliau-rcd** [comment]
Question for anyone, @Sean-McQ  or @rak-phillip ? Does the #6904 PR fix the actual issue or just fix the notification spam? Basically, if this is in 2.6.9 and we upgrade, will we still have websocket disconnect issues "under the hood"?

**Comment by gaktive** [comment]
For QA, https://github.com/rancher/dashboard/issues/6960 will be our fix for now to address this for 2.6.9 and 2.7.0 by removing growls.

We'll create a new growl behaviour for 2.7.1 involving an icon in the header bar for admins, so this ticket will be replaced.
